### DISCUSSION TITLE
ImageGlass Kobe 8.5.1.22

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1255] [General]
<img width="640" alt="ImageGlass 8.5.1.22" src="https://imageglass.org/upload/photo/release/8.5_1.jpg">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/8.5.1.22/total?color=%232A7C91&label=downloads&style=for-the-badge)](https://imageglass.org/download)

## 👐🏿 Support ImageGlass on Microsoft Store!
This is the paid version, you will need to pay a small amount after 7-day trial. If you buy ImageGlass in the Microsoft Store, you'll be supporting its development directly. You will get the convenience of fast, easy installation onto all of your Windows devices along with fully automatic, behind-the-scenes updates with all the newest features, improvements, and fixes.

<a href="https://www.microsoft.com/store/productId/9N33VZK3C7TH">
  <img height="75" src="https://upload.wikimedia.org/wikipedia/commons/f/f7/Get_it_from_Microsoft_Badge.svg" />
</a>

### 🌱 New:
- Added support for `QOI` image format.
- Added an option to play sound when a number of images change (#1199), thanks to @AutomationBoi.
- Added new shortcut to close ImageGlass: <kbd>Ctrl+W</kbd> (#1172).
- Added an option: _"Continue running background when the app is closed"_ to minimize the app window to system tray menu when user closes ImageGlass (#1196). To truly exit the app, right-click on ImageGlass tray icon, select Exit; or use _Main menu > Exit_ (<kbd>Shift+ESC</kbd>).
- Added an option: _"Start with OS"_ to auto-run ImageGlass when Windows starts and stay in system tray (#1205).

### 🔮 Improvements:
- Improved software security, all binary files are now signed (#97).
- Updated Magick.NET to 8.6.1.
- Added extensions icons for JXL, HEIF, CR3, AVIF, B64, QOI (#1183).

### 🐛 Bugfixes:
- Fixed an issue that the viewing image disappears when changing color channels (#1175).
- Fixed an issue that when editing image from the clipboard, ImageGlass ignores the editing app from _Settings > Edit_, always uses _MS Paint_ (#1139).
- On Windows 11, fixed an issue that the default editing function throws error because of the changes of Windows associated programs (#1160). ImageGlass is now using _MS Paint_ to edit by default on Windows 11.
- Fixed several issues that the image is corrupted while being saved when users navigate to other images or close the app by disabling the UI when the app is busy (#566, #1097, #1120, #1134).
- Fixed an issue that ImageGlass shows confusing error message when clicking on Unregister extensions button (#1174).

### 🥒 Checksums:

File | SHA-1
-- | --
ImageGlass_8.5.1.22_x64.msi | `6062fa31452e5b359b2d3893a6ae350d4a4c7fe8`
ImageGlass_8.5.1.22_x64.zip | `a04de0d9a833c562e2871a1834dabc204d1c76ca`
ImageGlass_8.5.1.22_x86.msi | `be49f886f8f81796bba872e8f0a10fd599f31b09`
ImageGlass_8.5.1.22_x86.zip | `3c9a00bc5c169251021d942f578987d606a43110`

## 🛣 Roadmap for ImageGlass 9
<img src="https://kontext.tech/api/flex/medias/obj-2290" alt="ImageGlass on .NET 6" height="50">

> We are planning to migrate ImageGlass to [.NET 6](https://devblogs.microsoft.com/dotnet/announcing-net-6/) from version 9 which is scheduled some time in 2022.
> If you are unable to install .NET 6, you can stay with ImageGlass 8.

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/8.5.1.22'>ImageGlass Kobe 8.5.1.22</a>.</em>

**Comment by loafylemon** [comment]
Bought Windows Store version to show my support, thanks for your hard work!
